test/acceptance/trade-area.js

```diff
@@ -46,6 +46,17 @@ describe('trade-area analysis', function() {
                 });
             });
         });
+
+        it('should return unique cartodb_ids', function (done) {
+            testHelper.getResult(tradeAreaDefinition, function(err, result) {
+                assert.ifError(err);
+                result.sort(function(a,b) {
```

Comment:
what about of using `Set` to get rid of duplicates and compare the lenght of both res and unique ids??:
```
const uniqueIds = [...new Set(result.map(r => r.cartodb_id))]
assert.equal(uniqueCartoDBIds.lenght, result.length)
```

Reply:
Will do, I didn't know about Set. Thanks!